### PR TITLE
dinar: update and change to netboot

### PR DESCRIPTION
**Pull request by tupakkatapa**

### CODE REVIEW

File: nixosConfigurations/dinar-ephemeral-alpha/secrets/default.yaml
Comment (jhvst) [inline nixosConfigurations/dinar-ephemeral-alpha/secrets/default.yaml:1]: removing secrets needs to be reflected in .sops.yaml

**Review by jhvst — CHANGES_REQUESTED** [review]

File: flake.nix [hunk 1]
```diff
@@ -236,6 +234,11 @@
                 outputs.overlays.modifications
               ];
             }
+            {
+              # Bootloader for x86_64-linux / aarch64-linux
+              boot.loader.systemd-boot.enable = true;
+              boot.loader.efi.canTouchEfiVariables = true;
+            }
```
[the remark this answers is not in the document]
Comment (tupakkatapa) [reply flake.nix:243]: Should this change be made straight to the kexecTree module?

[the remark this answers is not in the document]
Comment (jhvst) [reply flake.nix:243]: works for me

[the remark this answers is not in the document]
Comment (tupakkatapa) [reply flake.nix:243]: Forgot that it is in a different repo, making the changes here at least for now.

File: flake.nix [hunk 2]
```diff
@@ -182,13 +178,6 @@
                     super.makeModulesClosure (x // {allowMissing = true;});
                 })
               ];
-            }
-            {
-              # Bootloader for RaspberryPi 4
-              boot.loader.raspberryPi = {
-                enable = true;
-                version = 4;
-              };
```
Comment (jhvst) [inline flake.nix:167]: I think these might not be possible to be removed -- the RPi boots from the GPU and might thus require its own bootloader to be present.

**Comment by jhvst** [comment]
otherwise lgtm

**Comment by jhvst** [comment]
Realized yesterday that the README has to be updated, currently the dinar hosts are labeled as ISO images.

**Comment by tupakkatapa** [comment]
Pushed the missing README update straight to main: https://github.com/ponkila/homestaking-infra/commit/9d9b7e00a24b2e96702e1c43760e699389682bf8

**Pull request merged**